### DISCUSSION TITLE
Cannot login to locally hosted Vaultwarden when there is no internet connection

### DISCUSSION THREAD

**Discussion by lococola** [dani-garcia/vaultwarden#4711] [General]
I am running a locally hosted Vaultwarden instance on my Unraid server. I am using Pihole with NPM reverse proxy and it's working perfectly. But, if I unplug the router from the fiber, now I can't view my passwords. I can still open the VW login page, but after entering the password I get the following error:

NetworkError when attempting to fetch resource

Since I am running everything locally I should be able to access the vault even if the internet is down, right? Anyone know what the issue could be? Or where I can look for possible incorrect configurations?

**Comment by samuellhwarren** [comment]
May have something to do with the plugin. While you are using it locally, the plugin may still need a connection.

--

Samuel Warren

***@***.***

2537974789

________________________________
From: lococola ***@***.***>
Sent: Monday, July 8, 2024 9:47 AM
To: dani-garcia/vaultwarden ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dani-garcia/vaultwarden] Cannot login to locally hosted Vaultwarden when there is no internet connection (Discussion #4711)


I am running a locally hosted Vaultwarden instance on my Unraid server. I am using Pihole with NPM reverse proxy and it's working perfectly. But, if I unplug the router from the fiber, now I can't view my passwords. I can still open the VW login page, but after entering the password I get the following error:

NetworkError when attempting to fetch resource

Since I am running everything locally I should be able to access the vault even if the internet is down, right? Anyone know what the issue could be? Or where I can look for possible incorrect configurations?

**Reply by lococola** [reply]
Do you mean the browser plugin? I am getting the error when trying to connect using the webpage.

**Comment by samuellhwarren** [comment]
Oh then that's different. I have a similar setup and haven't attempted that option, unless it's maybe DNS related? You are using NPM, but try using just the ip address?

--

Samuel Warren

***@***.***

2537974789

________________________________

**Reply by lococola** [reply]
I'm afraid that also doesn't work. As in, if I connect using the IP address I do still get the login page, but then the error is that it requires a HTTPS connection. Hence the reverse proxy.
